### ISSUE TITLE
Touch ripple effect

### ISSUE THREAD

**Issue by 57r31** [Rezmason/matrix#13]
Should be looking nice on touchscreens

Like https://www.shadertoy.com/view/ll2GRD but with decay

**Comment by 57r31** [comment]
![5-1631115248952](https://user-images.githubusercontent.com/3975260/176361287-dd745236-5da2-403e-a183-ad510c45746d.png)

**Comment by 57r31** [comment]
ok, i did the base stuff
https://github.com/57r31/matrix
run with effect=ripples

**Comment by Rezmason** [comment]
I'll take a look tonight! Also any feedback you may have on the ability to make changes to this project is valuable in its own right 😊

**Comment by Rezmason** [comment]
Short version: I developed your proof of concept into a solid feature, with ripples propagating out from the click position. Up to five simultaneous ripples at a time. Adjusted their aspect ratio to appear circular, and also gave the glyphs a tint similar to what's in the _Resurrections_ shot, which brighten slightly in intensity where the ripples are. Also added webcam support, but that's a separate opt-in so no one is caught off-guard when the browser requests camera access.

Since a different kind of ripple already exists in the project, I think I'll rename the effect "mirror". 

I'll upload what I've got later tonight and link you to it. I'll also add an **Attribution** section to the readme, where contributors' names go. If that's all right with you, just let me know what moniker you'd like to be represented as.
